primitives/account/src/lib.rs

```diff
@@ -83,19 +83,21 @@ impl From<[u8; 20]> for EthereumSigner {
 
 impl From<ecdsa::Public> for EthereumSigner {
 	fn from(x: ecdsa::Public) -> Self {
-		let decompressed =
-			secp256k1::PublicKey::parse_slice(&x.0, Some(secp256k1::PublicKeyFormat::Compressed))
-				.expect("Wrong compressed public key provided")
-				.serialize();
+		let decompressed = libsecp256k1::PublicKey::parse_slice(
+			&x.0,
+			Some(libsecp256k1::PublicKeyFormat::Compressed),
+		)
+		.expect("Wrong compressed public key provided")
+		.serialize();
```

Comment:
@girazoki This is the only file with actual code changes. Should there be changes to use `overflowing` signatures?

Reply:
Not sure what that means